woocommerce/woocommerce-gutenberg-products-block
# Fix correct branding color

Fixes #4878  

#### Accessibility


### Screenshots

<table>
<tr>
<td>Before:
<br><br>

![#4878-before](https://user-images.githubusercontent.com/3323310/136719966-b42fa44e-6bd5-410b-857b-46fd27b817e9.png)
</td>
<td>After:
<br><br>

![#4878-after](https://user-images.githubusercontent.com/3323310/136719977-fb9995ff-01c2-4fd5-94d8-bdcccca9323e.png)
</td>
</tr>
</table>

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Create a test page.
2. Toggle the block inserter.
3. Scroll down to the WooCommerce blocks.
4. See the old WooCommerce color.
5. Check out this PR.
6. Refresh the page.
7. See the new WooCommerce color.
